src/sentry/workflow_engine/handlers/detector/stateful.py

```diff
@@ -298,6 +299,15 @@ def get_state_data(
             )
         return results
 
+    def get_all_triggered_group_keys(self) -> list[DetectorGroupKey]:
+        """Get all group keys that are currently triggered for this detector"""
+        from sentry.workflow_engine.models.detector_state import DetectorState
```

Comment:
DetectorState is already imported at the module level, could we just use that? might be a little more readable as `self.detector.detectorstate_set.filter(...)` as well, but i think that's a bit more pedantic.